src/cosalib/vultr.py

```diff
@@ -0,0 +1,23 @@
+from tenacity import (
+    retry,
+    stop_after_attempt
+)
+
+
+@retry(reraise=True, stop=stop_after_attempt(3))
+def vultr_run_ore(build, args):
+    """
+    Placeholder to upload a raw image to Vultr.
+    """
+    pass
+
+
+def vultr_run_ore_replicate(*args, **kwargs):
+    print("Images are not published to Vultr. This is a placeholder")
```

Comment:
I think for these two, it might be better to just e.g. `raise Exception("not implemented yet")` so that it's clear uploading/replication isn't supported yet.

Reply:
sure thing, added